src/test/java/com/hotel/api/book/mappers/book/UserInfoBuilderTest.java

```diff
@@ -0,0 +1,41 @@
+package com.hotel.api.book.mappers.book;
+
+import com.hotel.api.book.model.book.request.UserInfo;
+import com.hotel.service.common.Address;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class UserInfoBuilderTest {
+
+    @Mock
+    private AddressBuilder addressBuilder;
+    @InjectMocks
+    private UserInfoBuilder userInfoBuilder;
+
+    @Test
+    public void map() {
+        UserInfo userInfo = getUserInfo();
+        Address.Builder address = Address.newBuilder();
+        when(addressBuilder.map(userInfo.getAddress())).thenReturn(address.build());
+        com.hotel.service.common.UserInfo response = userInfoBuilder.map(userInfo);
+        assertThat(response).isNotNull();
+        assertThat(response.getFirstName()).isEqualTo("test");
+        assertThat(response.getLastName()).isEqualTo("123");
+        assertThat(response.getEmail()).isEqualTo("ash@gmail.com");
```

Comment:
1. Please add verify condition for when 
2.  Please add assertions for prefix, phone number. Please see subject class